### PR TITLE
#314: implemented switch for IMPORT statement for BigQuery dialect

### LINKED ISSUE #314: Implement a switch for BigQuery to enable IMPORT statement

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #314

### CODE REVIEW

File: src/main/java/com/exasol/adapter/dialects/bigquery/BigQuerySqlDialect.java [hunk 1]
```diff
@@ -48,7 +51,14 @@ protected RemoteMetadataReader createRemoteMetadataReader() {
 
     @Override
     protected QueryRewriter createQueryRewriter() {
-        return new BigQueryQueryRewriter(this, this.remoteMetadataReader, this.connection);
+        if (this.properties.containsKey(BIGQUERY_ENABLE_IMPORT_PROPERTY)
+                && "true".equalsIgnoreCase(this.properties.get(BIGQUERY_ENABLE_IMPORT_PROPERTY))) {
+            LOGGER.warning("Attention: IMPORT is activated for the BIGQUERY dialect. "
+                    + "Please be aware that it using IMPORT with this dialect requires disabling important security features and is therefore not recommended!");
```
Comment (redcatbear) [inline src/main/java/com/exasol/adapter/dialects/bigquery/BigQuerySqlDialect.java]:
```suggestion
                    + "Please be aware that using IMPORT with this dialect requires disabling important security features and is therefore not recommended!");
```

Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/adapter/dialects/bigquery/BigQuerySqlDialect.java]: fixed

File: doc/dialects/bigquery.md [hunk 2]
```diff
@@ -26,32 +28,27 @@ CREATE SCHEMA SCHEMA_FOR_VS_SCRIPT;
 
 The SQL statement below creates the adapter script, defines the Java class that serves as entry point and tells the UDF framework where to find the libraries (JAR files) for Virtual Schema and database driver.
 
-Please remember to check the versions of your JAR files after downloading driver. They can differ from the list below.
+List all the jars from Magnitude Simba JDBC driver.
```
Comment (redcatbear) [inline doc/dialects/bigquery.md]:
```suggestion
List all the JAR files from Magnitude Simba JDBC driver.
```

File: doc/dialects/bigquery.md [hunk 3]
```diff
@@ -101,6 +98,12 @@ If you need to use currently unsupported data types or find a way around known l
 ## Performance
 
 Please be aware that the current implementation of the dialect can only handle result sets with limited size (a few thousand rows).
+If you need to proceed a large amount of data, please, contact our support team. Another implementation of the dialect with a speed improvement(using IMPORT INTO) is available, but not documented and announced officially on our GitHub due to: 
+
+1. the complex installation process
+2. security risks (a user has to disable the drivers' security manager to use it)
+
+Please, try to avoid using IMPORT implementation if it is possible. Be aware of the security risks.
```
[the remark this answers is not in the document]
Comment (AnastasiiaSergienko) [reply doc/dialects/bigquery.md]: done

**Pull request merged**